playwright/test/selectors/images.ts

```diff
@@ -4,7 +4,7 @@ import { mobileModal, searchResultsContainer } from './search';
 export const searchImagesForm =
   'form[aria-describedby="images-form-description"]';
 export const colourSelectorFilterDropDown = `${searchImagesForm} button[aria-controls="images.color"]`;
-export const colourSelector = 'button[id="swatch-green"]';
+export const colourSelector = 'button[data-name="swatch-green"]';
```

Comment:
If it's just for test, `data-test-id` is often used across the codebase too (I guess it ensure we knows what it's for)